### ISSUE TITLE
[Request] Display cover image

### ISSUE THREAD

**Issue by GoogleCodeExporter** [AmatCoder/mednaffe#14]
```
An option to display a cover image or game snapshot of individual games would 
be very nice. Covers or images could be stored in a subdirectory or alongside 
the roms with matching file names (like Game.rom would match Game.jpg or 
Game.rom.jpg)
```

Original issue reported on code.google.com by `StarrySk...@gmail.com` on 2 Feb 2014 at 10:07

**Comment by polsvoice** [comment]
Yes, this would be very cool!

**Comment by AmatCoder** [comment]
Implemented on 2f0e40c commit.
